### ISSUE TITLE
Interesting paper about integrating software citation with Git

### ISSUE THREAD

**Issue by sdruskat** [force11/force11-sciwg#94]
Just stumbled across this: [*Automating Software Citation using GitCite*](https://arxiv.org/pdf/1902.00952.pdf) on arXiv.

The authors tackle software citation through citation functions (based on the [CoreCover algorithm](https://github.com/chenlica/alu01-corecover)) on nodes in a version of the repository tree, and couple git operations with citation operations (add/delete/modify).

While the system doesn't seem to fix existing issues with, e.g., automated retrieval of correct metadata for authorship, it is certainly an interesting take on citation automation.

**Comment by katrinleinweber** [comment]
Am I missing the link to their code in the PDF, or did they not publish that?!? The "Source" on arXiv doesn't contain anything besides the TeX files. https://chrome.google.com/webstore/search/GitCite comes up empty as well as of `Fri Jun 21 19:06:12 CEST 2019` 🤷‍♀

**Comment by sdruskat** [comment]
They've eaten their own dog food and provide links to commits in a listing (Listing 1) of a `citation.cite`file, but no usable (with reagrds to the Principles) citations :confused: ...

The repositories in question are:

- [CoreCover](https://github.com/chenlica/alu01-corecover)
- [GUI](https://github.com/thuwuyinjun/Data_citation_demo)

The actual implementations have can be cobbled together from the commit IDs and the repo URLs in the listing. Interesting approach, albeit not entirely in line with the Principles.

**Comment by katrinleinweber** [comment]
"GUI" is a 1 to 2 years-old _Data_ Citation Demo, not the Chrome extension the paper seems to mention. Maybe @thuwuyinjun & @chenlica can clarify? From where can I install your tool to test it?

**Comment by sdruskat** [comment]
According to the Listing, this commit is referenced as the GUI contribution: https://github.com/thuwuyinjun/Data_citation_demo/commit/2dd6813.

**Comment by katrinleinweber** [comment]
Yes, but apparently that's a different project, which they just happen to use as an example in the GitCite paper, or am I missing something? cc @Yanssie
